Tensile/KernelWriterAssembly.py

```diff
@@ -11748,17 +11748,32 @@ def openPrefetchAcrossPersistent(self, kernel, isOptNLL):
     if kernel["PersistentKernelAlongBatch"]:
       imod.addInst("s_mul_i32", sgpr(stmp), sgpr(stmp), sgpr("NumWorkGroups2"), "Total WG-0 x 1 x 2")
     imod.addInst("s_cmp_ge_u32", sgpr("SerialWorkGroupIter"), sgpr(stmp), "outside legal WG?")
-    imod.addInst("s_cbranch_scc1", self.getNamedLabel(label), "skip pf if OOB - last tile no PAP, go to pure OptNLL")
+    if useBufferOOB and kernel["PrefetchAcrossPersistentMode"] == 1:
```

Comment:
Do we need to check openPrefetchAcrossPersistent here?
It is already checked when setting useBufferOOB = True.